src/GafferImage/Display.cpp

```diff
@@ -324,7 +324,10 @@ class GafferDisplayDriver : public IECoreImage::DisplayDriver
 				memcpy( &backBuffer[0], &other.backBuffer[0], backBuffer.size() * sizeof( float ) );
 				dirty = (bool)other.dirty;
 				cachedTile = other.cachedTile;
-				cachedForDataCount = other.cachedForDataCount;
+				// cachedForDataCount is only really relevant to a particular Display, but we use
+				// this assignment when transferring tiles to a different driver used with a different
+				// display, so just reset it
```

Comment:
Perhaps something like this?

```suggestion
				// Tile copy constructor is used exclusively by `Display::setDriver( copy = true )`.
				// Reset data count to force a refresh (see corresponding reset in `setDriver()`).
```

Reply:
I've used these comments.

Reply:
And squashed into the main Display commit, now that we know we're going to go this way for this PR at least.